admin.py

```diff
@@ -13,7 +13,7 @@ class PartnerAdmin(admin.ModelAdmin):
     list_filter = ('name','as2_name')
     fieldsets = (
         (None, {
-            'fields': ('name', 'as2_name', 'email_address','target_url','subject','content_type')
+            'fields': ('name', 'as2_name', 'email_address','target_url','subject','content_type', 'confirmation_message')
```

Comment:
I didn't change the list page. I changed the fieldsets. When you define values in fieldsets you overwrite the default, which is to list all fields. After the default is overwritten you must manually define every field that you want to be able to view / modify on the admin page. If confirmation_message is not in fieldsets it does not show up in the admin page.